extensions/wikia/DesignSystem/DesignSystemHelper.class.php

```diff
@@ -72,7 +72,7 @@ private static function getCachedSvg( $name ) {
 			self::$svgCache[$name] = $xml;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)